setup.py

```diff
@@ -148,6 +148,7 @@ def main():
         entry_points='''
            [console_scripts]
            nipypecli=nipype.scripts.cli:cli
+           nipype_mprof=nipype.utils.profiler:main
```

Comment:
can we put this under `nipypecli`? it would be nice to have a single cli.

Reply:
I think we can even remove this executable.